react/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "scripts": {
+    "lint": "tsc --noEmit",
+    "test": "vtex-test-tools test",
+    "test:watch": "vtex-test-tools test:watch"
```

Comment:
```suggestion
```

I think `yarn test --watch` it's just about the same effect

Reply:
Indeed